controller/network/network.go

```diff
@@ -643,11 +643,13 @@ func (network *Network) CreatePath(srcR, dstR *Router) (*Path, error) {
 	if err != nil {
 		return nil, err
 	}
+	ingressId += "I"
```

Comment:
yah. i'll just say this onelast time..... :)

Reply:
Fixed. See above.